src/HotChocolate/Spatial/src/Types/GeoJsonGeometryEnumType.cs

```diff
@@ -0,0 +1,22 @@
+using static HotChocolate.Types.Spatial.GeoJsonGeometryType;
+
+namespace HotChocolate.Types.Spatial
+{
+    public sealed class GeoJsonGeometryEnumType : EnumType<GeoJsonGeometryType>
+    {
+        protected override void Configure(IEnumTypeDescriptor<GeoJsonGeometryType> descriptor)
+        {
+            descriptor.BindValuesExplicitly();
+
+            descriptor.GeoJsonName(nameof(GeoJsonGeometryType));
```

Comment:
We should create a constant class with the explicit type names and fill them in explicitly.